src/software/world/field.h

```diff
@@ -163,12 +170,26 @@ class Field
     Rectangle enemyDefenseArea() const;
 
     /**
-     * Gets the friendly half of the field
+     * Gets the enemy defense area extended to field boundary as a Rectangle.
+     *
+     * @return defense area of the enemy team
+     */
+    Rectangle enemyDefenseAreaToBoundary() const;
```

Comment:
Javadoc doesn't specify it's only expanded to the boundary in the y direction. Based on the name I'd expect it to expand in all directions (or at least +y, -y, -x).